etc/ide_setup/README.md

```diff
@@ -0,0 +1,59 @@
+# Intellij Setup
```

Comment:
I think it would be good to add the bit about adding the SDK in IntelliJ to the adoptjdk package. Also may want to add a troubleshooting bit about restarting the Maven Project if there are weird errors etc.